scene/resources/default_theme/default_theme.cpp

```diff
@@ -982,7 +982,7 @@ void make_default_theme(bool p_hidpi,Ref<Font> p_font) {
 	Ref<BitmapFont> default_font;
 	if (p_font.is_valid()) {
 		default_font=p_font;
-	} if (p_hidpi) {
+	} else if (p_hidpi) {
```

Comment:
No I think the change is fine. As I understand it, the behaviour of this method is (at least it was until now and the change keeps it):
- If p_font is a valid font, use it
- If it isn't, use the default font for either hidpi (if p_hidpi) or lodpi